app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -15,16 +15,18 @@ def initialize(ems, _options = nil)
       end
 
       def ems_inv_to_hashes
-        @data[:middleware_servers] = get_middleware_servers
+        # the order of the method calls is important here
```

Comment:
please elaborate why the order is important, otherwise the comment is not fully clear.